hack/jenkins/e2e.sh

```diff
@@ -398,30 +398,31 @@ case ${JOB_NAME} in
     : ${PROJECT:="kubernetes-jenkins"}
     ;;
 
-  # Sets up the GCE soak cluster weekly using the latest 1.1 release.
-  kubernetes-soak-weekly-deploy-gce-release-1.1)
-    : ${E2E_CLUSTER_NAME:="gce-soak-weekly-release-1.1"}
+  # Sets up the GCE soak cluster weekly using the latest 1.1 ci release.
+  kubernetes-soak-weekly-deploy-gce-1.1)
+    : ${E2E_CLUSTER_NAME:="gce-soak-weekly-1.1"}
```

Comment:
Acknowledged. gce-release-1.1 uses `E2E_CLUSTER_NAME="jenkins-gce-e2e-release-1.1"` so I was sort-of following that pattern.

We probably need to figure out a better way of refactoring these variables - it's starting to get a bit unwieldy in e2e.sh.